templates/CRM/Contribute/Form/Contribution/Confirm.tpl

```diff
@@ -223,14 +223,12 @@
 {crmRegion name="contribution-confirm-billing-block"}
        {if ($credit_card_number or $bank_account_number)}
         <div class="crm-group credit_card-group">
-            <div class="header-dark">
-            {if $paymentProcessor.payment_type & 2}
-                 {ts}Direct Debit Information{/ts}
-            {else}
-                {ts}Credit Card Information{/ts}
+            {if $paymentFieldsetLabel}
```

Comment:
I am sure the paymentFieldsetLabel includes the label 'Direct Debit Information' or 'Direct Debit Information'.